src/forklift/__main__.py

```diff
@@ -45,7 +45,7 @@
 from os.path import join
 
 log_location = join(abspath(dirname(__file__)), '..', 'forklift-garage', 'forklift.log')
-detailed_formatter = logging.Formatter(fmt='%(levelname).4s %(asctime)s %(module)10s:%(lineno)5s %(message)s',
+detailed_formatter = logging.Formatter(fmt='%(levelname)7s %(asctime)s %(module)10s:%(lineno)5s %(message)s',
```

Comment:
Can you give me more detail on what this does? Just curious. Allow 7 characters for the log level string?